cmd/nerdctl/container/container_commit.go

```diff
@@ -32,6 +32,7 @@ func CommitCommand() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:               "commit [flags] CONTAINER REPOSITORY[:TAG]",
 		Short:             "Create a new image from a container's changes",
+		Long:              "Create a new image from a container's changes. Use --format to choose between docker (default) or oci image format.",
```

Comment:
This explanation seems redundant.
Already covered in the help message of `--format`.